rednotebook/gui/clouds.py

```diff
@@ -157,9 +151,17 @@ def _get_cloud_body(self, cloud_words):
             self.link_index += 1
         return '\n'.join(html_elements)
 
+    @staticmethod
+    def _get_collated_word(word_and_freq):
+        word, freq = word_and_freq
+        return locale.strxfrm(word)
+
     def _get_tags_for_cloud(self, tag_count_dict, ignores):
-        return [(tag, freq) for (tag, freq) in tag_count_dict
-                if not any(pattern.match(tag) for pattern in ignores)]
+        tags_and_frequencies = [(tag, freq) for (tag, freq) in tag_count_dict
+                                if not any(pattern.match(tag) for pattern in ignores)]
+        tags_and_frequencies.sort(key=self._get_collated_word)
+        tag_display_limit = self.journal.config.read('tagDisplayLimit')
+        return tags_and_frequencies[-tag_display_limit:]
```

Comment:
I think we should present the most-used tags instead of the ones that come first in the alphabet.

Reply:
Ah, you are right - I must have looked at the function below. Fixed.